azerothcore/azerothcore-wotlk
# fix(DB/Quest): Post-partum Aggression

## CHANGES PROPOSED:
-  update smart script for Hemet Nesingwary and Stampy


## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3285
- Credits: el-druida



## TESTS PERFORMED:
- Apply sql and restart server
- .tele sholazar
- .quest add 12607
- complete quest A Mammoth Undertaking
- start quest Post-partum Aggression
- kill Broodmother Slivina and verify you got credit while riding the mammoth



## HOW TO TEST THE CHANGES:

- .tele sholazar
- .quest add 12607
- complete quest A Mammoth Undertaking
- start quest Post-partum Aggression
- kill Broodmother Slivina and verify you got credit while riding the mammoth

## KNOWN ISSUES AND TODO LIST:




## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
